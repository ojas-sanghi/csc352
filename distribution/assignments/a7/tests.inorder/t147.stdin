fe s bp no l zn h s n cd d tz h mq hq h d hx m v l z b b am l pb ek xp s fn ml r g o v n y yx m j ta q b g v y 
qv ea lf dp y e h e jb d s i dp r qb c tl e lz q a z xv m lo ls q n ru zg zy j g wh i j or 
e vd a w ti p hz gu fy h x pm m l bs 
z d zb lt rb ff yt w kk zo mm z x xs pk y xs c y i j x hx t h z vv md p d u e ny o xv 
t y r om xq h g m fk i kv h l a xs f v x l w m fx t j x tf iz zj f sa zz i w mu n yb iw h ao e iw l zm cy 
m si v li yx is b z vr kj oo b zo ay z gj j c yp j b v 
hl kk ap tn gl ec u b 
vb yw z k r a fl ta se p ce ui vw jc c qj b x c hw nz x s h o os wh a np n s 
f ru ql h f nf f c b y i i ju vl dn t h i l cp p xl jt 
ae m u an md cb ad gl ki go cg jz 
w d oa nz s e r s vz pd bt r xd fw b b rk j rw l r wu rg al sb h j kz se r po id 
o a z ws g o jp f e q od ov y g h hy qo vb ay ku hm hh uq q a w f x ep sj l 
r ct gu yz t 
o o ro c bl dy ah we tu s n gm ko yu d yz m wa a l r d ld sv qa qo ym pu zm as rc r te e sl hq m hp td n ob r y sd fc g 
j ic n za l yn e gc u nw k ui fj fa w y cy bq fz v g oc io jd nb gq a zg ud mg k tg xj fp z h jd nw br r ln fs 
d n br f rs ka nu f ul bw x cc jm n gy i t y wi ui e x qm wg sz ub qi j vg ih s u j zw yi h je cw 
fu e p lq b wi ct xb bn s c u g uy xm vp w f v q wy sy y t p 
z jn wm 
jr 
z m m ba a jm wz g v r yi zy l q r od a eh wy l rv o ym el k nv ww r t k h y vh nf h sp ag sc so 
ra is o z kq o cx ep l 
ux hm pn x g nb ms c y v a wj k e c se c s k ie pz v sa eq od h p e w 
l t m u y 
j kp zb f qz s tc r vw dl l tf m h ru s tf j bh uf 
dl v ln z v sn w qj ip h b kv xz nm ba ln k f e co nc kn 
i zf d dg js ls zo t b um jc ff he x l m n l q nh xn 
o 
p d bk y h wy e zs n yr tq b w v jf t i jl i a um nv lv s g fz ut r c ny ns b qg rv h 
o hr hl og u og dz w b pf br kh hv xt gs m ww bp q xq iy 
in w b nc lm ro uk gc ul ay ck pk q g hp yy ev yg a zt yy ao tb sf t 
v ra g a k z r aa e h wu lr qz hj qw lz di hh kc p fk r r px zu x cj ub lb f x cq q 
a q q na dg ta sf iv a o k ia zl a fr q q q p b lp xb 
sr xn px wz i r j a wj kw g sg 
xh m tx q y h ib h mh jf df xl z 
s o b sd i l ez lv zm tf c tf zf c lt c j jh i pq vg z nq gu n sc s nt o uj v rf qo f l z 
of e r y en g fy a nk c i br st m q ov on vr 
e r m fa uh rj x y h f t bn lk wy si k p dk r a ny 
es s e c dp o si s de f j s w h hu b gp de nn wz pa yp 
x f k rf i v vs db ok 
ew pt s 
pp 
t po oj i mk lu eu fj ae df m xq ft o nh m fa rr qc aq fk ws f td 
ja o q bl k de r 
x go po o s nb x q v 
g ux b ll w xv y rf qy cj aw l pi gh oq q l 
x v r s kj lb l ca np l ua od x o s yh z ys g a ko 
d j du z mg ki y w ol px k o vk yc fh xh w g ux r dj w h ik y bn x s h 
