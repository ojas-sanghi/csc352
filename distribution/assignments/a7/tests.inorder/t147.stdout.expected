a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
aa
ad
ae
ae
ag
ah
al
am
an
ao
ao
ap
aq
as
aw
ay
ay
ay
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
b
ba
ba
bh
bk
bl
bl
bn
bn
bn
bp
bp
bq
br
br
br
br
bs
bt
bw
c
c
c
c
c
c
c
c
c
c
c
c
c
c
c
c
c
ca
cb
cc
cd
ce
cg
cj
cj
ck
co
cp
cq
ct
ct
cw
cx
cy
cy
d
d
d
d
d
d
d
d
d
d
d
d
db
de
de
de
df
df
dg
dg
di
dj
dk
dl
dl
dn
dp
dp
dp
du
dy
dz
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
ea
ec
eh
ek
el
en
ep
ep
eq
es
eu
ev
ew
ez
f
f
f
f
f
f
f
f
f
f
f
f
f
f
f
f
f
f
fa
fa
fa
fc
fe
ff
ff
fh
fj
fj
fk
fk
fk
fl
fn
fp
fr
fs
ft
fu
fw
fx
fy
fy
fz
fz
g
g
g
g
g
g
g
g
g
g
g
g
g
g
g
g
g
g
g
gc
gc
gh
gj
gl
gl
gm
go
go
gp
gq
gs
gu
gu
gu
gy
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
h
he
hh
hh
hj
hl
hl
hm
hm
hp
hp
hq
hq
hr
hu
hv
hw
hx
hx
hy
hz
i
i
i
i
i
i
i
i
i
i
i
i
i
i
i
i
i
i
ia
ib
ic
id
ie
ih
ik
in
io
ip
is
is
iv
iw
iw
iy
iz
j
j
j
j
j
j
j
j
j
j
j
j
j
j
j
j
j
j
ja
jb
jc
jc
jd
jd
je
jf
jf
jh
jl
jm
jm
jn
jp
jr
js
jt
ju
jz
k
k
k
k
k
k
k
k
k
k
k
k
k
k
ka
kc
kh
ki
ki
kj
kj
kk
kk
kn
ko
ko
kp
kq
ku
kv
kv
kw
kz
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
l
lb
lb
ld
lf
li
lk
ll
lm
ln
ln
ln
lo
lp
lq
lr
ls
ls
lt
lt
lu
lv
lv
lz
lz
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
md
md
mg
mg
mh
mk
ml
mm
mq
ms
mu
n
n
n
n
n
n
n
n
n
n
n
n
n
na
nb
nb
nb
nc
nc
nf
nf
nh
nh
nk
nm
nn
no
np
np
nq
ns
nt
nu
nv
nv
nw
nw
ny
ny
ny
nz
nz
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
o
oa
ob
oc
od
od
od
od
of
og
og
oj
ok
ol
om
on
oo
oq
or
os
ov
ov
p
p
p
p
p
p
p
p
p
p
p
pa
pb
pd
pf
pi
pk
pk
pm
pn
po
po
po
pp
pq
pt
pu
px
px
px
pz
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
q
qa
qb
qc
qg
qi
qj
qj
ql
qm
qo
qo
qo
qv
qw
qy
qz
qz
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
r
ra
ra
rb
rc
rf
rf
rf
rg
rj
rk
ro
ro
rr
rs
ru
ru
ru
rv
rv
rw
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
s
sa
sa
sb
sc
sc
sd
sd
se
se
se
sf
sf
sg
si
si
si
sj
sl
sn
so
sp
sr
st
sv
sy
sz
t
t
t
t
t
t
t
t
t
t
t
t
t
t
ta
ta
ta
tb
tc
td
td
te
tf
tf
tf
tf
tf
tg
ti
tl
tn
tq
tu
tx
tz
u
u
u
u
u
u
u
u
ua
ub
ub
ud
uf
uh
ui
ui
ui
uj
uk
ul
ul
um
um
uq
ut
ux
ux
ux
uy
v
v
v
v
v
v
v
v
v
v
v
v
v
v
v
v
v
v
v
vb
vb
vd
vg
vg
vh
vk
vl
vp
vr
vr
vs
vv
vw
vw
vz
w
w
w
w
w
w
w
w
w
w
w
w
w
w
w
w
w
w
wa
we
wg
wh
wh
wi
wi
wj
wj
wm
ws
ws
wu
wu
ww
ww
wy
wy
wy
wy
wz
wz
wz
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
x
xb
xb
xd
xh
xh
xj
xl
xl
xm
xn
xn
xp
xq
xq
xq
xs
xs
xs
xt
xv
xv
xv
xz
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
yb
yc
yg
yh
yi
yi
ym
ym
yn
yp
yp
yr
ys
yt
yu
yw
yx
yx
yy
yy
yz
yz
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
za
zb
zb
zf
zf
zg
zg
zj
zl
zm
zm
zm
zn
zo
zo
zo
zs
zt
zu
zw
zy
zy
zz
